xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -186,7 +186,7 @@ platform.wiki.browse._actions.delete=Delete
 
 ## Classes
 WikiManager.WikiUserClass_membershipType_open=Open for any user to join
-WikiManager.WikiUserClass_membershipType_request=Any global user can request to join but an admin must accept
+WikiManager.WikiUserClass_membershipType_request=Any global user can request to join but an admin must accept it
```

Comment:
Sounds unnecessarily odd. Either say `must accept the request` (which might be too long) or leave it like it was (where it was implied and clear enough).

Reply:
Done.

Reply:
Looks like you forgot to make this change?

Reply:
The "it' is in the green :)

Reply:
Sorry, my bad. Fixed.